generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -211,7 +209,7 @@ module.exports = class extends BaseGenerator {
     this.fs.writeJSON(
       this.destinationPath(
         botName,
-        applicationSettingsDirectory,
+        this.applicationSettingsDirectory,
```

Comment:
This will now default to undefined instead of '.' - is there any impact as a result?

Reply:
Good catch - before this was extracted from `this.options` which is unchecked which is what I was intending to fix. I updated to default to `'.'`;